.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Lint and validate OpenAPI specs
+on:
+  - push
+  - pull_request_target
```

Comment:
See above for a suggestion on a potentially better definition of workflow trigger events.